testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java

```diff
@@ -272,9 +275,10 @@ private Environment getEnvironment()
                         .withFileSystemBind(testJar.getPath(), "/docker/test.jar", READ_ONLY)
                         .withFileSystemBind(cliJar.getPath(), "/docker/trino-cli", READ_ONLY)
                         .withCopyFileToContainer(forClasspathResource("docker/presto-product-tests/common/standard/set-trino-cli.sh"), "/etc/profile.d/set-trino-cli.sh")
+                        .withEnv("JAVA_HOME", jdkVersion.getJavaHome())
```

Comment:
I moved this env from `Standard` to keep both `JAVA_HOME` and `java` command path usage in single place